packages/mui-material/src/Rating/Rating.test.js

```diff
@@ -119,6 +120,41 @@ describe('<Rating />', () => {
     expect(container.querySelector('.customized')).to.have.tagName('label');
   });
 
+  it('should apply labelEmptyValueActive styles from theme', () => {
+    const theme = createTheme({
+      components: {
+        MuiRating: {
+          styleOverrides: {
+            labelEmptyValueActive: {
+              height: '120px',
+            },
+          },
+        },
+      },
+    });
+    const { container } = render(
+      <ThemeProvider theme={theme}>
+        <Rating value={null} />
+      </ThemeProvider>,
+    );
+
+    expect(container.querySelector(`.${classes.labelEmptyValueActive}`)).to.equal(null);
```

Comment:
No need to test if the `labelEmptyValueActive` class is getting applied or not to label in this test case.
```suggestion
```